### PR TITLE
Fix broken links.

### PR DESCRIPTION
**Pull request by jeroenleenarts**
### 🔗 Issue Links

_Provide all Jira tickets and/or Github issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅

File: docusaurus/docs/iOS/uikit/views/thread-header-view.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: ChatThreadHeaderView
 import ComponentsNote from '../../common-content/components-note.md'
 import Properties from '../../common-content/reference-docs/stream-chat-ui/chat-thread/chat-thread-header-view-properties.md'
 
-This component is responsible to display the information in the header of a thread. By default, it is rendered in the `navigationItem.titleView` of the [`ChatThreadVC`](../thread).
+This component is responsible to display the information in the header of a thread. By default, it is rendered in the `navigationItem.titleView` of the [`ChatThreadVC`](../../components/thread/).
```
Comment (alteral) [inline docusaurus/docs/iOS/uikit/views/thread-header-view.md:8]: e.g. this one

**Review by alteral — CHANGES_REQUESTED** [review]
Hey @jeroenleenarts, I just briefly looked at a couple of links and seems like most of them are still broken for me. Am I testing this wrong?

File: docusaurus/docs/iOS/uikit/views/thread-header-view.md (hunk 1 above)
Comment (alteral) [reply docusaurus/docs/iOS/uikit/views/thread-header-view.md:8]: it opens `https://github.com/GetStream/stream-chat-swift/blob/23f48f32193f9bfb508be33aab15dd88d9b63bf5/docusaurus/docs/iOS/components/thread`

but I think it should open `https://github.com/GetStream/stream-chat-swift/blob/23f48f32193f9bfb508be33aab15dd88d9b63bf5/docusaurus/docs/iOS/uikit/components/thread.md` 🤔

Comment (alteral) [reply docusaurus/docs/iOS/uikit/views/thread-header-view.md:8]: Hm, okay, will take a look when deployed 👍

**Pull request merged**